webpack/assets/javascripts/react_app/routes/RouterSelector.js

```diff
@@ -0,0 +1,5 @@
+export const selectRouterLocation = state => state.router.location;
```

Comment:
can you also add `export const selectRouter = state => state.router;`

Reply:
why if there's no use for it?

Reply:
Imo we should break the selectors to the smallest pieces, 
for external uses in other components,
also for a rare case when state.router doesn't exist
or for future usage with reselect